test/MorphoTokenEthereumTest.sol

```diff
@@ -191,9 +203,17 @@ contract MorphoTokenEthereumTest is BaseTest {
         deal(address(newMorpho), delegator1, amount1);
         deal(address(newMorpho), delegator2, amount2);
 
+        vm.expectEmit(address(newMorpho));
+        emit DelegationToken.DelegateeChanged(delegator1, address(0), delegatee);
+        vm.expectEmit(address(newMorpho));
+        emit DelegationToken.DelegatedVotingPowerChanged(delegatee, 0, amount1);
         vm.prank(delegator1);
         newMorpho.delegate(delegatee);
 
+        vm.expectEmit(address(newMorpho));
+        emit DelegationToken.DelegateeChanged(delegator2, address(0), delegatee);
+        vm.expectEmit(address(newMorpho));
+        emit DelegationToken.DelegatedVotingPowerChanged(delegatee, amount1, amount1 + amount2);
```

Comment:
Not sure that it's really useful to test events here, because it's already tested in testDelegate

Reply:
Perhaps, we could group all tests on events in a dedicated test called `testEventsEmited`, the point would be separation of concerns. The logic may be good but doesn't emit the test, so the logic test shouldn't fail if an event is not emited, instead the event test should fail. No strong feeling on this idea though.